README.md

```diff
@@ -1,90 +1,192 @@
 
-## Docker image for production
+## Available Scripts
+
+```bash
+yarn setup               # Install dependencies
+yarn dev                 # Run app in development mode
+yarn test                # Run tests in watch mode
+yarn test:coverage       # Run tests with coverage
+yarn build               # Create optimized production build
+yarn extract-translations # Extract English strings for Lokalise
+```
+
+---
+
+## Docker Image for Production
+
+To build a production Docker image:
 
 ```bash
 docker build \
 --build-arg VITE_GLIFIC_API_PORT=API-PORT \
---no-cache -t .
+--no-cache -t glific-frontend .
 ```
 
+Replace `API-PORT` with your backend API port number.
+
+---
+
 ## Localization
 
-Update translation base file (i.e. English)
+1. Extract base strings:
 
-```bash
-yarn extract-translations
-```
+   ```bash
+   yarn extract-translations
+   ```
 
-Once changed are pushed and merged into master it will be available in `lokalise.com` for translation.
+2. Once merged into `main`, strings will appear in [Lokalise](https://lokalise.com).
+3. Use `Download → Build only` in Lokalise to push translations.
+4. This will create a PR automatically in the frontend repo.
+5. It's recommended to **Preview** before building.
 
-After new strings are transalated. Use `Download >> Build only`. It will automatically create PR in this repo.
+---
 
-Note: Good to `Preview` before using `Build only`
+## Deploying Release on ECS with CD
 
-## Deploying release on ECS with CD
+- For AWS CodeBuild, use `buildspec.yml.sample` as your config file.
+- For Docker Hub and alternatives, use this GitHub Action:  
+  🔗 https://github.com/marketplace/actions/build-and-push-docker-images
 
-1. If you are using AWS codebuild for CD, use buildspec.yml.sample file content for creating and pushing docker image.
-2. For using and alternative repository like docker hub, click the link to see how we can [build and push docker](https://github.com/marketplace/actions/build-and-push-docker-images) images to docker hub.
+---
 
 ## Learn More
 
-### Glific
-
-- [Glific.org](https://glific.org/)
-- [One Pager](https://docs.google.com/document/d/1XYxNvIYzNyX2Ve99-HrmTC8utyBFaf_Y7NP1dFYxI9Q/edit?usp=sharing)
-- [Google Drive](https://drive.google.com/drive/folders/1aMQvS8xWRnIEtsIkRgLodhDAM-0hg0v1?usp=sharing)
-- [Product Features](https://docs.google.com/document/d/1uUWmvFkPXJ1xVMr2xaBYJztoItnqxBnfqABz5ad6Zl8/edit?usp=sharing)
-- [Blogs](https://chintugudiya.org/tag/glific/)
-- [Discord](https://discord.gg/scsrGUw)
+- 🌐 [Glific.org](https://glific.org/)
+- 📄 [One Pager](https://docs.google.com/document/d/1XYxNvIYzNyX2Ve99-HrmTC8utyBFaf_Y7NP1dFYxI9Q/edit?usp=sharing)
+- 📁 [Google Drive](https://drive.google.com/drive/folders/1aMQvS8xWRnIEtsIkRgLodhDAM-0hg0v1?usp=sharing)
+- 📝 [Product Features](https://docs.google.com/document/d/1uUWmvFkPXJ1xVMr2xaBYJztoItnqxBnfqABz5ad6Zl8/edit?usp=sharing)
+- 📰 [Blogs](https://chintugudiya.org/tag/glific/)
+- 💬 [Discord](https://discord.gg/scsrGUw)
```

Comment:
Can we put discord inside `chat with us` section,  similar to how we've done it in the backend? Including it inside `Learn More` doesn't seem to fit well contextually.